### ISSUE TITLE
implementing the API request to delete tags

### ISSUE THREAD

**Issue by Cutuchiqueno** [urschrei/pyzotero#56]
I am building a library to administer and query tags in a sophisticated way using pyzotero. I have a `sanitize` method which should change tags with upper case letters into lowercase versions. Unfortunately Zotero somehow memorizes a tag that was used once even if no item in the library has this tag. Since Zotero also does not distinguish between lowercase and uppercase tags the original uppercase tag is restored once I add its lowercase version. Thus it is not possible to change tags on the basis of capitalization and it is also not possible to remove all uppercase versions first and then add the lowercase versions.

I found out Zotero's web api does have a tag request for deleting tags from the library and not only removing them form items (https://www.zotero.org/support/dev/web_api/v3/write_requests#deleting_multiple_tags). Would you consider to implement this request type?

**Comment by urschrei** [comment]
Could you test this by uninstalling pyzotero, then doing `pip install https://github.com/urschrei/pyzotero/archive/dev.zip`? 
That should give you a `delete_tags(['tag a', 'tag b', 'tag c'])`(up to 50 tags) method. I'll merge it into master and cut a release if it's working correctly.

**Comment by Cutuchiqueno** [comment]
Hi,

thanks for this quick implementation! I tested it and it works perfect. Tags are completely removed from the tag library and not only from items so that versions of the same tag with different case can be added later on.

Just for curiosity and security reasons I also tested if tags are removed by substring match and fortunately they are not. Curiously enough the string that matches the tag which should be removed is case sensitive like it sould be in the Zotero API. I really wonder why the same does not comply with the way Tags are handled inside the Zotero GUI. So everything is great, thx.

The only small issue which might be a question of consistency within pyzotero is that the `add_tags` method add tags as a list of n parameters whereas the new `del_tags` method takes one parameter that is a list of n tags. I do not know if you want to change that but I wanted to mention it at least.

Thx
